platform-sdk/swirlds-unit-tests/core/swirlds-platform-test/build.gradle.kts

```diff
@@ -15,33 +15,34 @@
  */
 
 plugins {
-  id("com.swirlds.platform.conventions")
-  id("com.swirlds.platform.library")
-  id("com.swirlds.platform.benchmark-conventions")
-  id("org.gradle.java-test-fixtures")
+    id("com.swirlds.platform.conventions")
+    id("com.swirlds.platform.library")
```

Comment:
intentional?

Reply:
I assume that files were never touched after we introduced the new formatting and spotless.